### ISSUE TITLE
[Accessibility]: Disable Notepad on narrow screens

### ISSUE THREAD

**Issue by limitedmage** [sourcegraph/sourcegraph#36125]
### Audit type

Mobile device navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34198

### Problem description

We have decided that the Notepad feature should not be available to certain screen sizes. Notepad is a feature designed with extra screen real estate in mind, as a faster way to create Notebooks. Since everything that can be done with Notepad can also be done by creating the notebook directly, it is safe to disable when there is not enough space for it.

### Expected behavior

Notepad should be visible in screens under a certain size.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by sourcegraph-bot-2** [comment]
Heads up @benvenker @lguychard - the "team/search-product" label was applied to this issue.
